pkgs/by-name/ep/epson_201310w/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  rpmextract,
+  autoreconfHook,
+  file,
+  libjpeg,
+  cups,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "epson_201310w";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://download3.ebz.epson.net/dsc/f/03/00/15/64/77/6f825c323a9f4b8429e3f9f0f4e9c50b1b15f583/epson-inkjet-printer-201310w-${finalAttrs.version}-1.src.rpm";
+    hash = "sha256-596qMon5KmMHe7mh+ykdpWeazJOCGA6HmmtvdGheRmA=";
+  };
+
+  nativeBuildInputs = [
+    rpmextract
+    autoreconfHook
```

Comment:
Are `autoreconfHook` and `file` needed for anything?  Otherwise, we should remove them here (and from the argument at the top of the file) to reduce dependencies.